app/scripts/lib/message-manager.js

```diff
@@ -41,26 +33,19 @@ module.exports = class MessageManager extends EventEmitter{
       status: 'unapproved',
     }
     this.addMsg(msgData)
-    console.log('addUnapprovedMessage:', msgData)
```

Comment:
I always wondered if we wanted these logs around.

Someday it might be fun to add log-levels to the app, for convenient debugging.